packages/frontend/src/components/Backdrop/Modal.tsx

```diff
@@ -0,0 +1,31 @@
+import { Modal } from '@digdir/designsystemet-react';
+import type React from 'react';
+import { Backdrop } from '.';
+import styles from './backdrop.module.css';
+
+type ModalProps = Parameters<typeof Modal>[0];
+
+type Props = {
+  open: boolean;
+  children?: React.ReactChild | React.ReactChild[];
+} & ModalProps;
+
+/*
+ *  Wrapper component that adds a backdrop and fixes z-index
+ */
+export const ModalWithBackdrop = ({ children, open, ...props }: Props) => {
```

Comment:
https://storybook.designsystemet.no/?path=/docs/komponenter-modal--docs har backdrop, så du trenger ikke  denne

Reply:
Den ser jo ikke lik ut som vår backdrop.. Syns vi skal bruke samme backdrop komponent over alt.

Reply:
Enig i at vi burde style backdroppen likt, irriterende at den er forskjellig. Nesten sånn at det har lite verdi å bruke Modal-dialogen til Designsystemet her. Vi kan overstyre css for backdrop som brukes i DS slik at den blir lik vår egen? Føles unødvendig å plugge inn vår egen Backdrop på en egen komponent som allerede har det. 

Kan godt være egen oppgave, og tenker grovt:

1. Dialog er ikke brukt helt riktig iht Designsystemets doc
2. DS sin dialog er vesentlig forskjellig fra våre behov / vårt design
3. Det er raskere og tryggere å lage vår egen enn å tilpasse DS sin.